.gitignore

```diff
@@ -10,3 +10,4 @@ venv*/
 .python-version
 build/
 dist/
+.idea
```

Comment:
This probably belongs in your user-level `.gitignore` — there are no editor-specific ignore entries here.

Reply:
```suggestion
```